internal/qemu/machines.go

```diff
@@ -230,8 +230,9 @@ func (machines *Machines) createNew(ctx context.Context, qemuProvisioner provisi
 
 	if _, err = qemuProvisioner.Create(ctx, request,
 		provision.WithBootlader(true),
-		provision.WithUEFI(false), // UEFI doesn't work correctly on PXE timeout, as it drops to UEFI shell
+		provision.WithUEFI(false),
 		provision.WithLogWriter(logWriter),
+		provision.WithUEFI(machines.options.UEFIEnabled), // Note: UEFI doesn't work correctly on PXE timeout in QEMU, as it drops to UEFI shell
```

Comment:
Should this option be specified twice?
```suggestion
		provision.WithLogWriter(logWriter),
		provision.WithUEFI(machines.options.UEFIEnabled), // Note: UEFI doesn't work correctly on PXE timeout in QEMU, as it drops to UEFI shell
```

Reply:
Nope, was an oversight, fixed.